federfrei3/address-mover
# Address to moves migration

Changed:
- Removed Address Model
- Removed Address Migrations
- Removed Address Table
- Adjusted Associations & Dependencies
- Inserted Address-Columns to Move Tables
- Adjusted the Seeds
- Removed Address Routes
- Removed Address Controller
- Removed Policies
- Removed Address Views

How to check:
- Check models + new move model
- Check move seeds
- db-reset
